File: tests/fixtures/generic/vulnerable/taint_codeinj.cs
// @audit-fixture
// @rule: taint_codeinj
// @expected: detected
using Microsoft.CodeAnalysis.CSharp.Scripting;
public class ScriptController : Controller {
    public async Task<IActionResult> Run() {
        var result = await CSharpScript.EvaluateAsync(Request.Query["code"]);
        return Ok(result);
    }
}

File: tests/fixtures/generic/vulnerable/taint_codeinj.java
// @audit-fixture
// @rule: taint_codeinj
// @expected: detected
import javax.script.*;
public class EvalServlet extends HttpServlet {
    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws Exception {
        ScriptEngine engine = new ScriptEngineManager().getEngineByName("javascript");
        engine.eval(request.getParameter("code"));
    }
}
